entities.py

```diff
@@ -155,11 +155,14 @@ def use_card(self, card, exhaust, pile, enemies, target: "Enemy"=None) -> None:
         sleep(0.5)
         view.clear()
 
-    def draw_cards(self, middle_of_turn: bool=True, cards: int = 0):
+    def draw_cards(self, middle_of_turn: bool=True, cards: int = -1):
         """Draws [draw_cards] cards."""
         _ = middle_of_turn
-        if cards == 0:
+        if cards == -1:
```

Comment:
I wouldn't use a "special number" to indicate "use draw strength", because it's hard to read, and could hide a math error. Instead make it default to `None` and put

```python
if cards is None:
  cards = self.draw_strength
```

Reply:
Got it.